.circleci/config.yml

```diff
@@ -130,6 +130,9 @@ jobs:
           command: cargo install cargo-llvm-cov
       - run:
           name: Run and report tests and coverage (Minimum 67%)
+          environment:
+            # https://github.com/johnterickson/cargo2junit/issues/79
+            RUSTC_BOOTSTRAP: 1
```

Comment:
I wonder if we should run tests once without `RUSTC_BOOTSTRAP`, and then again with it just for the coverage report.

Reply:
I don't mean conditionally execute tests -- I mean always execute tests, then optionally re-run with structured output for the coverage report.

Reply:
Yup, exactly! Definitely agree that it's more of a "let's explore if there's an issue"